packages/storybook/stories/INTERNAL/ComplianceBanner.stories.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Checkbox, IStackStyles, IStackTokens, Stack } from '@fluentui/react';
+import { _ComplianceBanner as ComplianceBannerComponent } from '@internal/react-components';
+import { _useCompositeLocale } from '@internal/react-composites';
+import { Meta } from '@storybook/react/types-6-0';
+import React, { useState } from 'react';
+
+import { COMPONENT_FOLDER_PREFIX } from '../constants';
+import { hiddenControl } from '../controlsUtils';
+
+const ComplianceBannerStory = (): JSX.Element => {
+  const strings = _useCompositeLocale().strings.call;
+  const [callRecordState, setCallRecordState] = useState(true);
+  const [callTranscribeState, setCallTranscribeState] = useState(true);
+  return (
+    <Stack styles={containerStyles} tokens={stackTokens}>
+      <Stack horizontal tokens={stackTokens}>
+        <Checkbox
+          label="Recoding is in progress"
+          checked={callRecordState}
+          onChange={(ev?: unknown, isChecked?: boolean) => setCallRecordState(!!isChecked)}
```

Comment:
For those that may stumble upon this PR and have never seen the !! before,
it converts a truthy or falsey object to boolean. 
(e.g. 0, null, undefined, etc would be converted to false)